modules/ingest-attachment/build.gradle

```diff
@@ -17,12 +17,26 @@ esplugin {
 
 // this overwrites the 'versions' map from Elasticsearch itself, but indeed we want that -- we're interested in managing our dependencies
 // as we (and tika) demand, and are not interested in, say, having the same version of commons-codec as elasticsearch itself
+// when updating tika, please review it's parent pom : https://repo1.maven.org/maven2/org/apache/tika/tika-parent
+// and manually update the transitive dependencies here
 def versions = [
-  'tika'  : '2.9.1',
-  'pdfbox': '2.0.29',
-  'poi'   : '5.2.3',
-  'mime4j': '0.8.10',
-  'commonsCodec': '1.16.0'
+  'tika'  : '2.9.2',
+  'pdfbox': '2.0.31',
+  'poi'   : '5.2.5',
+  'sparsebitset' : '1.3', //poi dependency: https://repo1.maven.org/maven2/org/apache/poi/poi/
+  'mime4j': '0.8.11',
+  'commonsCodec': '1.16.1',
+  'slf4' : '2.0.10',
+  'xz' : '1.9',
+  'commonsIo' : '2.15.1',
+  'juniversalchardet' : '1.0.3', //TODO: update this version with the updated group
+  'tagsoup' : '1.2.1',
+  'jempbox' : '1.8.17',
+  'xmlbeans' : '5.1.1', //TODO: where does this come from ?
```

Comment:
Looks like it ought to be 5.2.0 because `org.apache.poi:poi-ooxml` (used for parsing microsoft docs) pulls it in.

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/108144/commits/a7a25c11ec9710cd848f0f9a6b1cb1157dd208c2